strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-nuget-layout-provider/src/main/java/org/carlspring/strongbox/nuget/file/AssemblyTargetFrameworkAdapter.java

```diff
@@ -0,0 +1,73 @@
+package org.carlspring.strongbox.nuget.file;
+
+import com.google.common.base.Joiner;
+import com.google.common.base.Strings;
+
+import java.text.MessageFormat;
+import java.util.EnumSet;
+import javax.xml.bind.annotation.adapters.XmlAdapter;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class AssemblyTargetFrameworkAdapter
+        extends XmlAdapter<String, EnumSet<Framework>>
+{
+
+    private static final Logger LOG = LoggerFactory.getLogger(AssemblyTargetFrameworkAdapter.class);
+    private static final String FRAMEWORKS_DELIMETER = ", ";
+
+    public AssemblyTargetFrameworkAdapter()
+    {
+    }
+
+    public String marshal(EnumSet<Framework> frameworks)
```

Comment:
@gokhankuyucak does this method used anywhere? I think that we should drop all unused methods across the PR changes, except `getters/setters` which is needed for serialization/deserialization.

Reply:
+1